compiler/ballerina-lang/src/main/java/io/ballerina/projects/PackageResolution.java

```diff
@@ -402,7 +413,8 @@ void resolve(ImportModuleRequest importModuleRequest, PackageDependencyScope sco
             }
 
             PackageOrg packageOrg = importModuleRequest.packageOrg();
-            List<PackageName> possiblePkgNames = getPossiblePackageNames(importModuleRequest);
+            List<PackageName> possiblePkgNames;
```

Comment:
Do we want to break this line into two here? Looks unnecessary to me.

Reply:
Yes, we can remove it.

Reply:
Fixed with b139f6d